concordium-cis2/src/cis2_client.rs

```diff
@@ -0,0 +1,433 @@
+//! CIS2 client is the intermediatory layer between any contract and
+//! CIS2 comliant contract.
+//!
+//! # Description
+//! It allows the contract to abstract away the logic of calling the
+//! CIS2 contract for the following methods
+//! - `supports_cis2` : Calls [`supports`](https://proposals.concordium.software/CIS/cis-0.html#supports)
+//! - `operator_of` : Calls [`operatorOf`](https://proposals.concordium.software/CIS/cis-2.html#operatorof)
+//! - `balance_of` : Calls [`balanceOf`](https://proposals.concordium.software/CIS/cis-2.html#balanceof)
+//! - `transfer` : Calls [`transfer`](https://proposals.concordium.software/CIS/cis-2.html#transfer)
```

Comment:
A small example of how to set up a client and call `balance_of` and `transfer` would be nice here.

Reply:
done